packages/component-library/src/index.tsx

```diff
@@ -4,6 +4,7 @@ export { default as DatePicker } from './DatePicker';
 export { default as Fieldset } from './Fieldset';
 export { default as FilePicker } from './FilePicker';
 export { default as Footer } from './Footer';
+export { default as HeroImage } from './HeroImage';
```

Comment:
Nice! I missed adding the base components I made here; I'll handle in a separate PR